quickstart/101-devtest-labs/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "resource_group_location" {
+  default     = "eastus"
```

Comment:
Should provide variable types.

Reply:
Agreed, @TomArcherMsft would you please add `type = string` for this variable?